### PR TITLE
clarify the all images help text

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@legionus ptal, should be a very quick review :)

File: pkg/oc/admin/prune/images.go [hunk 1]
```diff
@@ -145,7 +145,7 @@ func NewCmdPruneImages(f *clientcmd.Factory, parentName, name string, out io.Wri
 	}
 
 	cmd.Flags().BoolVar(&opts.Confirm, "confirm", opts.Confirm, "If true, specify that image pruning should proceed. Defaults to false, displaying what would be deleted but not actually deleting anything. Requires a valid route to the integrated Docker registry (see --registry-url).")
-	cmd.Flags().BoolVar(opts.AllImages, "all", *opts.AllImages, "Include images that were not pushed to the registry but have been mirrored by pullthrough.")
+	cmd.Flags().BoolVar(opts.AllImages, "all", *opts.AllImages, "Include images are imported from external registries.")
```
Comment (juanvallejo) [inline pkg/oc/admin/prune/images.go]: I think it'd read a bit better with `that` after `Include`: `Include images that are...`

Comment (bparees) [reply pkg/oc/admin/prune/images.go]: err, yeah. totally left a word out. thanks @juanvallejo 

updated.

**Comment by legionus** [comment]
@bparees may be better to write like this:
> Include images that were imported from external registries, all the mirrored objects from them will be removed from the integrated registry.

?

**Comment by bparees** [comment]
@legionus revised with some additional tweaks, ptal.

**Pull request merged**